metadata_service/proxy/neo4j_proxy.py

```diff
@@ -162,7 +163,7 @@ def _exec_table_query(self, table_uri: str) -> Tuple:
         OPTIONAL MATCH (application:Application)-[:GENERATES]->(tbl)
         OPTIONAL MATCH (tbl)-[:LAST_UPDATED_AT]->(t:Timestamp)
         OPTIONAL MATCH (owner:User)<-[:OWNER]-(tbl)
-        OPTIONAL MATCH (tbl)-[:TAGGED_BY]->(tag:Tag)
+        OPTIONAL MATCH (tbl)-[:TAGGED_BY]->(tag:Tag{tag_type: $tag_type})
```

Comment:
@danwom , if I don't change this line, you should get both badge and tag together through the API which FE needs to change the logic to distinguish these two based on the tag_type. Does it work for you?

Reply:
```
tag_fields = {
    'tag_type': fields.String,
    'tag_name': fields.String
}

table_writer_fields = {
    'application_url': fields.String(attribute='application_url'),
    'name': fields.String,
    'id': fields.String,
    'description': fields.String  # Optional
}



table_detail_fields = {
    'database': fields.String,
    'cluster': fields.String,
    'schema': fields.String,
    'table_name': fields.String(attribute='name'),
    'table_description': fields.String(attribute='description'),  # Optional
    'tags': fields.List(fields.Nested(tag_fields)),  # Can be an empty list
    # Can be an empty list
    'table_readers': fields.List(fields.Nested(table_reader_fields)),
    # Can be an empty list
    'owners': fields.List(fields.Nested(user_fields)),
    # Can be an empty list
    'columns': fields.List(fields.Nested(column_fields)),
    # Can be an empty list
    'watermarks': fields.List(fields.Nested(watermark_fields)),
    'table_writer': fields.Nested(table_writer_fields),  # Optional
    'last_updated_timestamp': fields.Integer,  # Optional
    'source': fields.Nested(source_fields),  # Optional
    'is_view': fields.Boolean  # Optional
}```

Reply:
this is the original payload.

Reply:
also @jinhyukchang

Reply:
As a last resort getting badges and tags in the `tag` field would be workable, but can't we put it in a new field `badges` using the following?

```
  OPTIONAL MATCH (tbl)-[:TAGGED_BY]->(tag:Tag{tag_type: 'default'})
  OPTIONAL MATCH (tbl)-[:TAGGED_BY]->(badge:Tag{tag_type: 'badge'})
  ...
  collect(distinct tag) as tag_records,
  collect(distinct badge) as badge_records,
```

Reply:
this will change table API which we need to see if it impacts atlas backend.